src/forms/TextInput.jsx

```diff
@@ -82,20 +107,16 @@ export const TextInput = props => {
 		e.target.setCustomValidity(validityMessage);
 	};
 
+	const optionalInputProps = {};
 	// WC-158
 	// Only add a `value` prop if it is defined.
 	// Workaround for IE11 support (see ticket)
 	if (value !== undefined) {
-		other.value = value;
+		optionalInputProps.value = value;
 	}
 
-	// Character limits should be a "soft" limit.
```

Comment:
removed this as its already being pulled of of "other" when we initially deconstruct it